src/it/scala/performanceanalysis/ComponentRegistrationTest.scala

```diff
@@ -40,9 +39,11 @@ class ComponentRegistrationTest extends IntegrationTestBase with Protocol {
       And("""I registered a component with id "parsConfigComp"""")
       registerComponent("parsConfigComp")
 
-      When("""I do a POST with {"regex" : "+d", "metric-key" : "a-numerical-metric"} to /components/parsConfigComp/metrics on the Administrator port""")
+      When(
+        """I do a POST with {"regex" : "+d", "metric-key" : "a-numerical-metric", "value-type": "string"}
+          |to /components/parsConfigComp/metrics on the Administrator port""".stripMargin)
       val path = "/components/parsConfigComp/metrics"
-      val data = """{"regex" : "+d", "metric-key" : "a-numerical-metric"}"""
+      val data = """{"regex" : "+d", "metric-key" : "a-numerical-metric", "value-type": "string"}"""
```

Comment:
Personally, I like string interpolation in testcase descriptions: 
`When(s"I do a POST with $data to $path on the Administrator port)`, but I don't know what is good practice here.